pkg/executor/test/showtest/show_test.go

```diff
@@ -807,7 +807,7 @@ func TestAutoRandomWithLargeSignedShowTableRegions(t *testing.T) {
 	tk.MustExec("drop table if exists t;")
 
 	tk.MustExec("create table t (a bigint unsigned auto_random primary key clustered);")
-	tk.MustExec("set @@global.tidb_scatter_region=1;")
+	tk.MustExec("set @@session.tidb_scatter_region=1;")
```

Comment:
Why here still use "1"?

Reply:
since using enum indicate three options for `tidb_scatter_region` .
https://github.com/pingcap/tidb/blob/5986baca1e4d75b7d7959fa03a1f0f91cbc4e8c0/pkg/sessionctx/variable/sysvar.go#L778
1. `0` equal `""`
2. `1` equal `"table"`
3. `2` equal `"global"`

suggesting use `"table"` instead of `1`?

Reply:
I know，what I mean is since we use enum, so can we only accept string input? 0,1,2 is not good for understand.